backend/src/main/java/de/neuefische/backend/model/Location.java

```diff
@@ -24,4 +24,5 @@
     private String picture;
     private String homepage;
     private List<Event> upcomingEvents;
-}
+    private boolean myLocation;
```

Comment:
I would use a list holding all userId's watching this location

```suggestion
    private List<String> watchedBy;
```